HelpSource/Classes/TGrains.schelp

```diff
@@ -14,23 +14,23 @@ classmethods::
 method::ar
 
 argument::numChannels
-Number of output channels.
+The number of output channels.
 
 argument::trigger
 At each trigger, the following arguments are sampled and used as
 the arguments of a new grain. A trigger occurs when a signal
-changes from non-positive to positive value.
+changes from non-positive to a positive value.
 
 If the trigger is audio rate then the grains will start with
 sample accuracy.
 
 argument::bufnum
-The index of the buffer to use. It must be a one channel (mono)
+The index of the buffer to use. It must be one channel (mono)
```

Comment:
The other changes are innocuous, but this one is incorrect. The indefinite article "a" is not optional here.